shell/browser/api/electron_api_base_window.h

```diff
@@ -195,6 +195,9 @@ class BaseWindow : public gin_helper::TrackableObject<BaseWindow>,
   std::string GetAlwaysOnTopLevel();
   void SetWindowButtonVisibility(bool visible);
   bool GetWindowButtonVisibility() const;
+  void SetWindowButtonPosition(absl::optional<gfx::Point> position);
+  absl::optional<gfx::Point> GetWindowButtonPosition() const;
+  // Deprecated.
   void SetTrafficLightPosition(const gfx::Point& position);
   gfx::Point GetTrafficLightPosition() const;
```

Comment:
We should follow our typical deprecation cycle pattern here and wrap these functions in `lib/browser/api/base-window.ts` with a `deprecate.warnOnce` call, something like:

```js
BaseWindow.prototype.setTrafficLightPosition = function (position) {
  deprecate.warnOnce('setTrafficLightPosition', 'SetWindowButtonPosition')
  BaseWindow._setTrafficLightPosition(position);
}
```

for both deprecated functions